docker/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Docker for server
+
+## build
+The docker is easyly build using the following command - By default it will build the docker for the **develop** branch:
+
+```
+ docker build . -t iotagent-turtle 
+```
+
+If we want to build other branch, we could define the branch we want to use this way:
+```
+ docker build . -t iotagent-turtle --build-arg VERSION=integration-cb-httpserver
```

Comment:
Following the previous comment, I suggest changing VERSION by BRANCH

Reply:
Ok